SimpleWAWS/Models/Site.cs

```diff
@@ -126,6 +126,14 @@ public string ContentDownloadUrl
             }
         }
 
+        public string FunctionContentDownloadUrl
+        {
+            get
+            {
+                return string.Concat("https://tryappservice.azure.com/api/resource/getfunctionappcontent/", SiteName, "^", ResourceGroupName);
```

Comment:
I think we should move the tryappservice.azure.com portion to a SimpleSetting with that as the default value . This way it can be setup in other environments as well. I know there are other places that use this harcoding but I have recently setup a test environment where if you want to test it .. stuff like this will break

Reply:
Done